src/app/zap-templates/templates/app/MatterIDL.zapt

```diff
@@ -61,40 +61,102 @@
     {{/unless}} {{asLowerCamelCase name~}}
     {{~#if isArray~}} [] {{~/if}} = {{code}};
   {{/chip_server_cluster_attributes}}
-  {{#chip_cluster_commands}}
-  {{#if arguments}}
+  {{#if (is_server side)}}
+  {{#all_incoming_commands_for_cluster name 'server'}}
+  {{#zcl_command_arguments}}
 
-  request struct {{asUpperCamelCase name}}Request {
-    {{#chip_cluster_command_arguments}}
+{{#first}}
```

Comment:
This indentation is very hard to read... Why are we not indenting things so it's clear what's inside what?